### PR TITLE
Adding 2016

### PR DESCRIPTION
**Pull request by get-itips**
Based on current support https://spmtreleasescus.blob.core.windows.net/install/default.htm
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2258
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2323

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/migrate-to-sharepoint-online.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ To learn more about the which tool to use, see the table below.
...
-|SharePoint Migration Tool|The SharePoint Migration Tool (SPMT) can migrate your files from SharePoint on-premises document libraries, lists or regular files shares.</br></br>Download either the current release or the latest public preview: </br></br>[SharePoint Migration Tool (current release)](https://spmtreleasescus.blob.core.windows.net/install/default.htm) or </br>[SharePoint Migration Tool Public Preview](https://spmtreleasescus.blob.core.windows.net/betainstall/default.htm)|Ideal for migrating SharePoint Server 2010 & 2013 environments.|
+|SharePoint Migration Tool|The SharePoint Migration Tool (SPMT) can migrate your files from SharePoint on-premises document libraries, lists or regular files shares.</br></br>Download either the current release or the latest public preview: </br></br>[SharePoint Migration Tool (current release)](https://spmtreleasescus.blob.core.windows.net/install/default.htm) or </br>[SharePoint Migration Tool Public Preview](https://spmtreleasescus.blob.core.windows.net/betainstall/default.htm)|Ideal for migrating SharePoint Server 2010, 2013 & 2016 environments.|
```
Comment (JohanFreelancer9) [inline migration/migrate-to-sharepoint-online.md:38]:
```suggestion
|SharePoint Migration Tool|The SharePoint Migration Tool (SPMT) can migrate your files from SharePoint on-premises document libraries, lists or regular files shares.</br></br>Download either the current release or the latest public preview: </br></br>[SharePoint Migration Tool (current release)](https://spmtreleasescus.blob.core.windows.net/install/default.htm) or </br>[SharePoint Migration Tool Public Preview](https://spmtreleasescus.blob.core.windows.net/betainstall/default.htm)|Ideal for migrating SharePoint Server 2010, 2013, and 2016 environments.|
```

File: migration/migrate-to-sharepoint-online.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ To learn more about the which tool to use, see the table below.
...
 |Mover|Service for cloud to cloud migration. To learn more, see: [Mover](https://mover.io/)|Ideal for migrating data from other cloud service providers.|
-|PowerShell|Using the PowerShell version of the SPMT. See [Migrate to SharePoint using PowerShell](https://docs.microsoft.com/sharepointmigration/overview-spmt-ps-cmdlets).|SPMT is ideal for migrating SharePoint Server 2010 & 2013 environments via PowerShell. |
+|PowerShell|Using the PowerShell version of the SPMT. See [Migrate to SharePoint using PowerShell](https://docs.microsoft.com/sharepointmigration/overview-spmt-ps-cmdlets).|SPMT is ideal for migrating SharePoint Server 2010, 2013 & 2016 environments via PowerShell. |
```
Comment (JohanFreelancer9) [inline migration/migrate-to-sharepoint-online.md:40]:
```suggestion
|PowerShell|Using the PowerShell version of the SPMT. See [Migrate to SharePoint using PowerShell](https://docs.microsoft.com/sharepointmigration/overview-spmt-ps-cmdlets).|SPMT is ideal for migrating SharePoint Server 2010, 2013, and 2016 environments via PowerShell. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**